ProcessLib/TwoPhaseFlowWithPP/TwoPhaseFlowWithPPLocalAssembler-impl.h

```diff
@@ -65,6 +64,9 @@ void TwoPhaseFlowWithPPLocalAssembler<
     auto Mlpc = local_M.template block<cap_pressure_size, cap_pressure_size>(
         cap_pressure_matrix_index, cap_pressure_matrix_index);
 
+    NodalMatrixType laplace_factor;
+    laplace_factor.setZero(nonwet_pressure_size, cap_pressure_size);
```

Comment:
@endJunction NodalMatrixType holds the size of [ShapeFunction::NPOINTS] times [ShapeFunction::NPOINTS], while nonwet_pressure_size is equal to ShapeFunction::NPOINTS, so is cap_pressure_size.